.travis.yml

```diff
@@ -22,12 +22,10 @@ jobs:
         - make setup/dep setup/travis code/check test/unit
     - stage: compile
       script:
-        - make setup/dep setup/travis image/build TAG=$(git rev-parse --short ${TRAVIS_COMMIT})
+        - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
+        - make setup/dep setup/travis image/build TAG=$BRANCH
     - stage: push
       script:
         - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
-        - export TAG=$(git rev-parse --short ${TRAVIS_COMMIT})
         - docker login --password "$QUAY_PASSWORD" --username "$QUAY_USERNAME" quay.io
-        - make setup/dep setup/travis image/build/push TAG=$TAG
-        - docker tag quay.io/integreatly/$OPERATOR_NAME:$TAG quay.io/integreatly/$OPERATOR_NAME:$BRANCH
-        - docker push quay.io/integreatly/$OPERATOR_NAME:$BRANCH
\ No newline at end of file
+        - make setup/dep setup/travis image/build/push TAG=$BRANCH
```

Comment:
Seems no point in adding all the individual commit builds to quay.io now. These were only here to support the way we had been doing releases of integreatly.

As this is now it will build an image for each branch/tag on the origin repo and push it quay.io.

Reply:
Actually, on second thoughts i'm going to undo this for now. This has the potential to cause problems for out 1.2.x releases of integreatly. We can apply this change when we are done with the 1.2 stream.